astropy/modeling/fitting.py

```diff
@@ -30,6 +30,7 @@
 import warnings
 
 from functools import reduce
+from pkg_resources import iter_entry_points
```

Comment:
Is this an *optional* dependency of Astropy or not? If so, we treat it like we do with Scipy.